system/Commands/Database/ShowTableInfo.php

```diff
@@ -145,6 +147,22 @@ public function run(array $params)
         $this->showDataOfTable($tableName, $limitRows, $limitFieldValue);
     }
 
+    private function showDBConfig(): void
+    {
+        $data = [[
+            'hostname' => $this->db->hostname,
+            'database' => $this->db->getDatabase(),
+            'username' => $this->db->username,
```

Comment:
Access to protected property `username`. Although `port`, `hostname` are also protected, there is no error for them

Reply:
There is a magic getter `__get()`.